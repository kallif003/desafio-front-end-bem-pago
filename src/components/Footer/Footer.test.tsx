import * as React from "react"
import { render } from "@testing-library/react"

import { Footer } from "./index"

describe("Header components", () => {
	it("Renders Header properly", () => {
		const { container } = render(<Footer>Test</Footer>)
		expect(container.firstChild).toMatchSnapshot()
	})
})
