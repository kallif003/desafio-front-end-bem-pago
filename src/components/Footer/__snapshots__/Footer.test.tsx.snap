// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer components Renders Footer properly 1`] = `
<footer
  class="sc-bdvvtL DriOi"
>
  Test
</footer>
`;
